⚡️ Allow `mgm-hurry hyperopt_show_epoch --fthypt` to use a `fthypt_name`

taina0407 opened the pull request:
Update mgm-hurry hyperopt_show_epoch to allow fthypt_name param to select fthypt file

taina0407 committed:
Allow hurry hyperopt_show_epoch using fthypt_name

Update mgm-hurry hyperopt_show_epoch to allow fthypt_name param to select fthypt file

Rikj000 reviewed (changes requested):
Thank you for your efforts! But I wonder if this brings any benefit to be honest?

We already have the `--fthypt` modifier in place, if you add that to your command then `mgm-hurry` will launch an interactive prompt to pick a `.fthypt` file of choice. Which seems better to me because then you don't need to manually check for the filename anymore.

However I think that it could be a benefit to extend upon the existing modifier instead in the following way:
- Scrap `--fthypt_name`
- If `--fthypt <fthypt-filename>.fthypt` is provided then skip the interactive prompt and simply use the `.fthypt` file that got provided in the command.
- If `--fthypt` or `--fthypt true` is passed launch the interactive prompt to pick a `.fthypt` file of choice
- If `--fthypt <incorrect-fthypt-filename>` is passed, print a warning that the filename could not be parsed/found and launch the interactive prompt

taina0407 commented:
> Thank you for your efforts! But I wonder if this brings any benefit to be honest?
> 
> We already have the `--fthypt` modifier in place, if you add that to your command then `mgm-hurry` will launch an interactive prompt to pick a `.fthypt` file of choice. Which seems better to me because then you don't need to manually check for the filename anymore.
> 
> However I think that it could be a benefit to extend upon the existing modifier instead in the following way:
> 
> * Scrap `--fthypt_name`
> * If `--fthypt <fthypt-filename>.fthypt` is provided then skip the interactive prompt and simply use the `.fthypt` file that got provided in the command.
> * If `--fthypt` or `--fthypt true` is passed launch the interactive prompt to pick a `.fthypt` file of choice
> * If `--fthypt <incorrect-fthypt-filename>` is passed, print a warning that the filename could not be parsed/found and launch the interactive prompt

Actually I wanna implement that, but was worry about Docs update. I will make update as that proposal

taina0407 committed:
Update hurry hyperopt_show_epoch fthypt param

Update hyperopt_show_epoch --fthypt param to accept boolean and file name as value

Rikj000 reviewed (approved):
Tested & looks good to me! :smile: 
Thank you for your contribution :mechanical_arm: 
I'll make sure to update the documentation for it with my next push to `development`
